plugin/mutate/source/dextool/plugin/mutate/backend/test_mutant/package.d

```diff
@@ -12,14 +12,16 @@ module dextool.plugin.mutate.backend.test_mutant;
 import core.time : Duration, dur;
 import logger = std.experimental.logger;
 import std.algorithm : map, filter, joiner, among, max;
-import std.array : empty, array, appender;
+import std.array : empty, array, appender, replace;
 import std.datetime : SysTime, Clock;
 import std.datetime.stopwatch : StopWatch, AutoStart;
 import std.exception : collectException;
 import std.format : format;
 import std.random : randomCover;
 import std.traits : EnumMembers;
 import std.typecons : Nullable, Tuple, Yes, tuple;
+import std.file : readText;
+import std.json;
```

Comment:
move this import to the state IncreaseFilePrio. Also change the import to those specific symbols that are used. I think it is JSONValue and JSONOptions.